gtsfm/two_view_estimator.py

```diff
@@ -47,14 +68,124 @@ def __init__(
             matcher: matcher to use.
             verifier: verifier to use.
             inlier_support_processor: post-processor that uses information about RANSAC support to filter out pairs.
+            bundle_adjust_2view: boolean flag indicating if bundle adjustment is to be run on the 2-view data.
             eval_threshold_px: distance threshold for marking a correspondence pair as inlier during evaluation
                 (not during estimation).
         """
         self._matcher = matcher
         self._verifier = verifier
         self.processor = inlier_support_processor
+        self._bundle_adjust_2view = bundle_adjust_2view
         self._corr_metric_dist_threshold = eval_threshold_px
 
+    @classmethod
+    def triangulate_two_view_correspondences(
```

Comment:
I think it might be more useful to have a triangulation util with variable number of cameras, can be used in DA as well.

Reply:
I think having a GTSAM API as Frank suggested is the best way forward. In that case we don't need a util function right now.